Doc/library/inspect.rst

```diff
@@ -511,10 +511,25 @@ Retrieving source code
    returned as a single string.  An :exc:`OSError` is raised if the source code
    cannot be retrieved.
 
+   If *object* is a wrapper function (that is to say has a :attr:`__wrapped__`
+   field, :func:`getsource` will follows the chain of :attr:`__wrapped__`
+   attributes returning the last object in the chain using :func:`unwrap`
+
    .. versionchanged:: 3.3
       :exc:`OSError` is raised instead of :exc:`IOError`, now an alias of the
       former.
 
+.. function:: findsource(object)
+
+   Return the entire source file and starting line number for an object.
```

Comment:
I'd be explicit that the function is returning a tuple, maybe by starting like this and describing both elements:

    Return a tuple ``(lines, lnum)`` where *lines* is ...